notebooks/MAST/K2/K2_Lightcurve/k2_lightcurve.ipynb

```diff
@@ -0,0 +1,334 @@
+    "# The transits are in there too, but you'd need to clean the light curve before you see them.\n",
+    "ax.set_xlim(2920., 2950.)\n",
+    "ax.set_ylim(0.1e7, 0.2e7)\n",
+    "\n",
+    "# Adjust the left margin so the y-axis label shows up.\n",
+    "plt.subplots_adjust(left=0.15)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Understanding Light Curve Flags\n",
+    "\n",
+    "The table of information contains more than just timestamps and fluxes.  In particular, there is a column of flags associated with every timestamp that indicate a number of warnings and conditions associated with that measurement.  Not every flag is worth excluding from your analysis: you should always make your own decision.  A summary of the flags can be found [here](https://archive.stsci.edu/kepler/manuals/archive_manual.pdf#page=19)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# First we need to read in the array of cadence quality flags, let's do\n",
+    "# that now.\n",
+    "with fits.open(fits_file, mode=\"readonly\") as hdulist:\n",
+    "    qual_flags = hdulist[1].data['SAP_QUALITY']"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now let's plot the full time series, but this time we'll overplot those points that have a quality flag greater than zero in red."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Start figure and axis.\n",
+    "fig, ax = plt.subplots()\n",
+    "\n",
+    "# Plot the timeseries in black circles.\n",
+    "ax.plot(k2_bjds, pdcsap_fluxes, 'ko')\n",
+    "\n",
+    "# Locate quality flags greater than zero.\n",
+    "where_gt0 = np.where(qual_flags > 0)[0]\n",
+    "\n",
+    "# Overplot the fluxes with quality flags greater than zero in red.\n",
+    "ax.plot(k2_bjds[where_gt0], pdcsap_fluxes[where_gt0], 'ro')\n",
+    "\n",
+    "# Let's zoom in on the x-axis and y-axis.\n",
+    "ax.set_xlim(2920., 2950.)\n",
+    "ax.set_ylim(0.1e7, 0.2e7)\n",
+    "\n",
+    "# Let's label the axes and define a title for the figure.\n",
+    "fig.suptitle(\"TRAPPIST-1 Light Curve - Campaign 12\")\n",
+    "ax.set_ylabel(\"PDCSAP Flux (e-/s)\")\n",
+    "ax.set_xlabel(\"Time (TBJD)\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Intringuingly, some of the largest outliers in the positive flux direction are flagged: are these bad measurements that should be excised from the time series?  Finding out the quality flag value and converting the value to its consitutent bit masks to understand why these points are flagged would be the first step.  We encourage you to do this as a follow-up to this tutorial."
```

Comment:
"Intringuingly" I'm not sure if you meant "interestingly" or "intriguingly."

Reply:
typo fixed